packages/main/src/models/conversation.ts

```diff
@@ -109,9 +109,7 @@ export function getPresentableElements({
         isRead: cache
           .getFlags(latestEdit ? latestEdit.revision.message.id : message.id)
           .includes("\\Seen"),
-        isStarred: cache
-          .getFlags(latestEdit ? latestEdit.revision.message.id : message.id)
-          .includes("\\Flagged"),
+        isStarred: cache.getFlags(message.id).includes("\\Flagged"),
```

Comment:
This still does not work. In this context `message` is the first revision of a presentable. If the presentable has been edited it is possible that one of the edit messages has been flagged, and in that case `isStarred` should be `true`. But this line does not check flags on edit messages. I recommend removing this property from this pull request, and getting back to it in the per-presentable starring feature.